### DISCUSSION TITLE
Using vue cmp inside grapesjs modal

### DISCUSSION THREAD

**Discussion by iabhiyaan** [GrapesJS/grapesjs#3774] [Q&A]
Hello @artf, Is there any way that we can use vue component inside grapesjs modal ?

**Comment by artf** [comment]
Just send the rendered HTMLElement in [Modal.open](https://grapesjs.com/docs/api/modal_dialog.html#open)
```js
const el = document.querySelector('#vue-element-id');
editor.Modal.open({
 title: 'Title',
 content: el,
})
```

**Reply by iabhiyaan** [reply]
It means first we should render vue component.

**Reply by artf** [reply]
Correct, you can also render the vue component inside a hidden div, if it's something that should be used only by the modal.

**Reply by iabhiyaan** [reply]
Thanks @artf.
